spec/lib/rmt/cli/products_spec.rb

```diff
@@ -488,6 +488,10 @@
     end
 
     describe 'success' do
+      it 'has mirroring enabled Installer Updates repositories by default' do
+        expect(expected_output).to include('Installer-Updates')
```

Comment:
This expectation is a bit fragile, it's asserting over a local variable and not from the result of some operation. In fact, no operation from the class under test is executed. Taking a look the other examples in the same group, we could have something like:

```suggestion
        expect { described_class.start(argv) }
          .to output(/Installer-Updates/).to_stdout
          .and output('').to_stderr
```

However, testing only the output could still be prone to false positives: the command generating an output containing a string like `'Installer-Updates'` doesn't ensure the repository has the attribute `mirroring_enabled == true` in the `repository` database table.

Maybe we can think of a more robust test scenario and I would suggest implementing it for the `RMT::SCC` class, since the logic for enabling 'Installer-Updates` repos by default is there. It seems `spec/lib/rmt/cli/products_spec.rb` only has expectation for text output. What do you think?

Reply:
Ok, I used the same output as suggested in the issue but it didn't feel quite right.
I will implement the test for `RMT::SCC` class then.

Reply:
Done.